bin/pygrb/pycbc_pygrb_results_workflow

```diff
@@ -206,8 +206,17 @@ logging.info("Offtrials: %s", [f.storage_path for f in offtrial_files])
 bank_file = os.path.join(start_rundir, args.bank_file)
 bank_file = _workflow.resolve_url_to_file(bank_file)
 
-# File instances of the input injection files
+# Sanity check and File instances of the input injection files
 inj_sets = wflow.cp.get_subsections('injections')
+inj_sets.sort()
+eff_secs = wflow.cp.get_subsections('pygrb_efficiency')
+eff_secs.sort()
+if eff_secs != inj_sets:
+    err_msg = "Each [injections] subsection requires a "
+    err_msg += "corresponding [pygrb_efficiency] subsection. "
+    err_msg += "[injections] subsections found: %s. "
+    err_msg += "[pygrb_efficiency] subsections found: %s. "
+    logging.error(err_msg, inj_sets, eff_secs)
```

Comment:
Hmm, there might be use cases where these subsections wouldn't be needed (e.g. running only one injection set, or just running a NS injection set ... maybe some new theory proves next year that NSBH GRBs definitely don't exist), but if it's better to enforce this, I won't object

Enforcing the two lists be equal is possibly not what you want, there could be other subsections in `efficiency` that are not injection names. I think you really want to check that inj_sets is a subset of eff_secs.
```
if set(inj_sets).issubset(eff_secs):
```

Reply:
Okay, I see your point.  I may want to make more efficiency plot versions, so I need at least enough `eff_secs` to cover the `inj_sets`.  I will make this change shortly.